killbill-integration-tests/core/usage/test_capacity.rb

```diff
@@ -29,6 +28,7 @@ def teardown
 
 
     def test_basic
+      aggregate_mode
```

Comment:
Is the assumption that these tests will be run using both modes, `AGGREGATE_MODE` and `DETAIL_MODE ` ?

Reply:
It would be nice indeed to run the exact same tests in both modes and have some sort of switch in the tests based on the configured mode (so that we don't end up duplicating the tests themselves):

```
...
if aggregate_mode?
  check_usage_invoice_item(...)
  check_invoice_item_detail(...)
else
  check_usage_invoice_item(...)
  check_invoice_item_detail(...)
end
...
```